data.go

```diff
@@ -79,56 +79,58 @@ func PopulateBeers() []model.Beer {
 			Created: time.Date(2017, time.October, 24, 22, 27, 0, 0, time.UTC),
 		},
 		model.Beer{
-			ID: 	 7,
-			Name:	 "XX Lager",
+			ID:      7,
+			Name:    "XX Lager",
 			Brewery: "Cuahutemoc Moctezuma",
-			Abv: 4.2,
+			Abv:     4,
```

Comment:
Ha, we should be using doubles for this not ints :D

Reply:
(don't feel like you need to fix this here - let's do it in a separate PR)

Reply:
yeah, well. I didn't add this in the first place, but it was annoying me with error `constant 4.2 truncated to integer` when I updated the brand and was testing my code after recent merge. Anyways, I agree this shouldn't be in this PR, better create a new issue for this and new PR. I am changing it back as requested.